search-resources-api/src/main/java/no/unit/nva/search/SearchTicketAuthHandler.java

```diff
@@ -61,8 +60,6 @@ protected List<MediaType> listSupportedMediaTypes() {
 
     @Override
     protected void validateRequest(Void unused, RequestInfo requestInfo, Context context) throws ApiGatewayException {
-        if (Objects.isNull(requestInfo.getUserName())) {
-            throw new UnauthorizedException();
-        }
+        requestInfo.getUserName();
```

Comment:
Do we have/need a check here? Are we guaranteed that requests are authorized when they make it this far?

Reply:
when you ask for UserName, an unauthorized exception is raised if there is no user name.
I can explain later....